contracts/Governance/Governor.sol

```diff
@@ -296,22 +295,27 @@ contract Governor {
         Proposal storage proposal = proposals[proposalId];
         require(
             msg.sender == guardian ||
+                // Allows anyone to cancel a proposal if the voting power of the
+                // proposer dropped below the threshold after the proposal was
+                // submitted.
                 token.getPriorVotes(proposal.proposer, sub256(block.number, 1)) <
                 proposalThreshold(),
-            "Governor::cancel: proposer above threshold"
+            "Governor::cancel: cannot cancel unless proposer is below threhsold"
         );
 
         proposal.canceled = true;
-        for (uint256 i = 0; i < proposal.targets.length; i++) {
-            timelock.cancelTransaction(
-                proposal.targets[i],
-                proposal.values[i],
-                proposal.signatures[i],
-                proposal.calldatas[i],
-                proposal.eta
-            );
-        }
 
+        if (_state == ProposalState.Queued) {
```

Comment:
Note that this is a semantic change. In the current version, you can cancel proposals that are `Queued` or `Expired`, whereas with this change you can only cancel `Queued` transactions. Of course, `Expired` proposals can't be executed anyway, so canceling them is mainly an cosmetic operation (unless the admin of the Timelock is changed).

I'm not sure how I feel about this change tbh. I don't think the gas efficiency concern brought up by ToB is particularly relevant. We'll remove the mention of this in our report.

Reply:
Hmm, I'm fine with reverting the change. I thought that since `proposal.canceled = true` is still run, this wouldn't actually change the semantics. :eyes:

Reply:
Since proposals aren't marked as cancelled in the `timelock`, it would only make a difference when upgrading the timelock admin